pkg/statistics/handle/bootstrap.go

```diff
@@ -285,7 +298,7 @@ func (h *Handle) initStatsHistogramsLite(ctx context.Context, cache statstypes.S
 }
 
 func (h *Handle) initStatsHistograms(is infoschema.InfoSchema, cache statstypes.StatsCache) error {
-	sql := "select  /*+ ORDER_INDEX(mysql.stats_histograms,tbl)*/ HIGH_PRIORITY table_id, is_index, hist_id, distinct_count, version, null_count, cm_sketch, tot_col_size, stats_ver, correlation, flag, last_analyze_pos from mysql.stats_histograms order by table_id"
+	sql := initStatsHistogramsSQLGen(false)
```

Comment:
This seems different from the old one. Could you please explain why we need to remove is_index=1 here?

Reply:
If you search the caller of the `initStatsHistogramSQLGen` you will see that now `initStatsHistogramsLite`, `initStatsHistograms`, and `initStatsHistogramsConcurrency` use the same SQL gen.
Actually, the SQL is not changed for `initStatsHistogramsLite` and `initStatsHistograms`. It just aligned the `initStatsHistogramsConcurrency` with the other two.

Reply:
Got it!